carbon-design-system/carbon
# Fix(Dropdown): Made the titleText required property

Closes #17843

This PR makes the `titleText` an optional property in the propTypes, since we are only showing `titleText` conditionally.

#### Changelog

**Changed**

- Made the `titleText` optional in propTypes.

#### Testing / Reviewing

- Remove the `titleText` property from <Dropdown /> in _"packages/react/src/components/Dropdown/Dropdown.stories.js"_
- Navigate to the **packages/react** directory using `cd packages/react` and run `yarn storybook`
- Check the console in browsers's developer tools
